src/ui/public/timepicker/kbn_global_timepicker.js

```diff
@@ -18,6 +18,7 @@ UiModules
 
   return {
     template: toggleHtml,
+    replace: true,
```

Comment:
We've shied away from using `replace: true` because it has been deprecated for so long, but it seems that it [won't ever be removed in angular 1.x](https://github.com/angular/angular.js/commit/eec6394a342fb92fba5270eee11c83f1d895e9fb#commitcomment-6621180) so hooray!